spark-cassandra-connector/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
if this is for testing purposes, maybe you could add file appender so that the logs are not so transient